Segment-Taplytics/Classes/SEGTaplyticsIntegrationFactory.m

```diff
@@ -22,6 +21,12 @@ + (instancetype)instance
     return sharedInstance;
 }
 
++ (instancetype)initializeTaplyticsManually
+{
+    static SEGTaplyticsIntegrationFactory *sharedInstance;
```

Comment:
Think about what this is doing.

You aren't creating a factory here. You defined a variable, which is nil and returned it/nil.

You still need to creat a facttory that calls your `initWithoutSettings` initializer.